tox.ini

```diff
@@ -66,3 +66,7 @@ python =
     3.9 = py39
     3.10 = py310
     3.11 = py311
+
+[flake8]
+max-line-length = 127
```

Comment:
Considering it is set [here](https://github.com/understandable-machine-intelligence-lab/Quantus/blob/2f19b31e01f6788ff5ae999232e110821f3af261/tox.ini#L45), a lot of files are using the 127 line length (I tried to run black and noticed it). Can we change this as a technical debt in a specific PR? Otherwise this PR will be big and with a different context.

Reply:
agree -- this is definitely a topic for separate PR :)